crypto/evp/legacy_sha.c

```diff
@@ -216,7 +216,7 @@ const EVP_MD *EVP_shake##bitlen(void)                                          \
         NID_shake##bitlen,                                                     \
         0,                                                                     \
         bitlen / 8,                                                            \
-        EVP_MD_FLAG_XOF,                                                       \
+        EVP_MD_FLAG_XOF | EVP_MD_FLAG_DIGALGID_ABSENT,                         \
```

Comment:
```suggestion
        (EVP_MD_FLAG_XOF | EVP_MD_FLAG_DIGALGID_ABSENT),                       \
```

Reply:
@vdukhovni not sure why these parentheses are useful but...